src/index.ts

```diff
@@ -14,6 +14,13 @@ export interface Options<Context> {
   withScope?: ExceptionScope<Context>
   captureReturnedErrors?: boolean
   forwardErrors?: boolean
+  reportError?: (res) => boolean
+}
+
+function containsIgnoredError(res, ignoredErrors) {
```

Comment:
It seems like this code is unused, could you confirm @mattfwood?

Reply:
My mistake, this was from an earlier approach I considered. Just removed it.